packages/core/src/Input.js

```diff
@@ -40,7 +40,7 @@ const Input = styled.input`
   margin: 0;
 
   ::placeholder {
-    color: ${themeGet('colors.gray')};
+    color: ${themeGet('palette.text.light')};
```

Comment:
Is there any difference between `themeGet` vs `getPaletteColor`?

Reply:
Fixed to use `getPaletteColor`